src/components/Subscription/SubscriptionBanner.css

```diff
@@ -57,3 +49,61 @@
 .SubscriptionBanner .dg.Paragraph.Paragraph--tiny  {
   font-size: 15px;
 }
+
+.SubscriptionBanner > .dcl.close {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: 8px;
+  background-color: #f1f6fe;
```

Comment:
Please add `z-index: 10;`

This will fix this problem:

![Screen Shot 2022-02-09 at 13 25 37](https://user-images.githubusercontent.com/279449/153245196-e62382ef-cb2e-40bd-af79-464afe693a36.png)
m